.github/workflows/dependabot-automerge.yml

```diff
@@ -0,0 +1,23 @@
+
+name: Automerge Dependabot
+
+on:
+  pull_request:
+    branches: [ main ]
```

Comment:
Out of interest, do we need to specify the branches?

Reply:
I don't think it is strictly needed because we know that most PRs are raised against main, but it just restricts when the action runs in the unlikely event of a PR towards a different branch.  I also think it makes it easily readable, but happy to remove it as well.